packages/types/src/payment-types.ts

```diff
@@ -54,12 +32,60 @@ export interface IFeeReferenceBasedCreationParameters extends IReferenceBasedCre
 }
 
 /** Parameters to create a request with "any to erc20" payment network */
-export interface IAnyToErc20CreationParameters extends IFeeReferenceBasedCreationParameters {
-  network?: string;
+export interface IAnyToErc20CreationParameters extends ICreationParametersAnyToAny {
   acceptedTokens?: string[];
-  maxRateTimespan?: number;
 }
 
+/** Interface to create a payment network  */
+export interface IPaymentNetworkCreateParameters<T = any> {
+  id: PAYMENT_NETWORK_ID;
+  parameters: T;
+}
+
+export type PaymentNetworkCreateParameters =
+  | {
+      id:
+        | ExtensionTypes.PAYMENT_NETWORK_ID.ERC20_PROXY_CONTRACT
+        | ExtensionTypes.PAYMENT_NETWORK_ID.ERC20_FEE_PROXY_CONTRACT
+        | ExtensionTypes.PAYMENT_NETWORK_ID.ETH_FEE_PROXY_CONTRACT
+        | ExtensionTypes.PAYMENT_NETWORK_ID.ETH_INPUT_DATA;
+      parameters: ExtensionTypes.PnFeeReferenceBased.ICreationParameters;
```

Comment:
For `ERC20_PROXY_CONTRACT` and `ETH_INPUT_DATA`, shouldn't it be `PnReferenceBased` instead of `PnFeeReferenceBased`?

Reply:
Well spotted, thanks!